src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -9,42 +9,64 @@
 
 helps['login'] = """
 type: command
-short-summary: Log in to Azure.
+short-summary: Log into Azure.
```

Comment:
I think "log in to" has been the convention in Azure CLI for some time, but I am not a native English speaker and I am definitely open to it.

Reply:
I agree with @jiasli here. The action is to "log in" and I believe should be "log in to".

Reply:
You are correct.

"Login, spelled as one word, is only a noun or an adjective. For example, the information you use to sign in to your email is your login (noun), and the page where you sign in is the login page (adjective). Log in is two words when it functions as a verb. For example, you log in with your login information."